### DISCUSSION TITLE
Microsoft Authenticator - Azure AD MFA implementation

### DISCUSSION THREAD

**Discussion by screwfox** [ente-io/ente#1018] [Enhancements]
**Is your feature request related to a problem? Please describe.**
My work utilise Azure AD and Microsoft Authenticator  n

**Describe the solution you'd like**
To be able to scan a Microsoft Authenticator QR Code and utilise ente-authenticator in place of the Microsoft one for the  Azure AD MFA implementation

**Comment by genterminl** [comment]
I was led to Ente Auth specifically as a potential substitute for Microsoft Authenticator, but it doesn't recognize the MS QR code.  Is there any hope for this to work, or are there any other suggestions?

**Reply by marknoble** [reply]
Yeah, me too. Instead of a QR code, it can give you a numeric code, as well as a URI like this:

https://mobileappcommunicator.auth.microsoft.com/activatev2/{redacted-numeric}/{redacted-string}

I'm looking to replace iOS/Android authenticators with ones that run on Windows.
